src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public Figure getRandomFigure(){

```

Comment:
The method `getRandomFigure` should return a random figure object. Currently, it's an empty method which might lead to a `NullPointerException` if used. You should implement the logic to return a random `Figure` object.